src/libraries/System.Data.Odbc/tests/CommandBuilderTests.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Data.Odbc.Tests
 {
     public class CommandBuilderTests : IntegrationTestBase
     {
-        [Fact(Skip = "Native dependencies missing in CI. See https://github.com/dotnet/runtime/issues/20097.")]
+        [ConditionalFact]
```

Comment:
This will no longer skip the test right? I think if we don't pass any conditions it runs it.

Reply:
Oh, never mind, I didn't see the SkipTestException from the base class, disregard 😄